yamllint/cli.py

```diff
@@ -125,6 +130,7 @@ def run(argv=None):
         sys.exit(-1)
 
     max_level = 0
+    underlined = '\033[4m'  # w/a - see colorama issue #38
```

Comment:
This is ugly hack, but if you consider changing file name style to bright (\e[1m) - this can be done like in the orher places.